pkg/MetaMorphoAction.ts

```diff
@@ -2,119 +2,219 @@ import { BigNumberish } from "ethers";
 import { MetaMorpho__factory } from "types";
 import { MarketAllocationStruct, MarketParamsStruct } from "types/src/MetaMorpho";
 
-export type MetaMorphoCall = string;
+const METAMORPHO_IFC = MetaMorpho__factory.createInterface();
 
-export class MetaMorphoAction {
-  private static METAMORPHO_IFC = MetaMorpho__factory.createInterface();
+export type MetaMorphoCall = string;
 
+export namespace MetaMorphoAction {
   /* CONFIGURATION */
 
-  static setCurator(newCurator: string): MetaMorphoCall {
-    return MetaMorphoAction.METAMORPHO_IFC.encodeFunctionData("setCurator", [newCurator]);
+  /**
+   * Encodes a call to a MetaMorpho instance to set the curator.
```

Comment:
can be a bit clearer, but not big take
```suggestion
   * Encodes a call to a MetaMorpho vault to set the curator.
```